pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
   "toml~=0.10",
   "httpx~=0.23",
   "pydantic~=1.10",
+  "websockets~=11.0",
```

Comment:
Make this dependency opt-in, eg. `pip install ariadne-codegen[subscriptions]`

Reply:
Hm I'm not sure if it's the right way. `AsyncBaseClient` still depends on `websockets` even when no subscription was generated.

Reply:
It should handle `websockets` not being installed. Usual approach is handling the import error and importing placeholder object that implements the subset of API but raises not implemented error from its methods.